library/alloc/src/collections/linked_list.rs

```diff
@@ -109,7 +107,6 @@ impl<T: fmt::Debug> fmt::Debug for IterMut<'_, T> {
 /// (provided by the `IntoIterator` trait). See its documentation for more.
 ///
 /// [`into_iter`]: struct.LinkedList.html#method.into_iter
```

Comment:
I can't make a suggestion because it's up two lines (what is github thinking?) but you can use `LinkedList::into_iter()` here too.